setup.py

```diff
@@ -35,6 +35,7 @@
             'local = regrowl.bridge.local:LocalNotifier',
             'subscribe = regrowl.bridge.subscribe:SubscribelNotifier',
             'udp = regrowl.bridge.udp:UDPNotifier',
+            'netcent = regrowl.bridge.netcent:NetCenter'
```

Comment:
For OCD reasons, could you sort this alphabetically and end the line in a "," ? :)